src/test/java/software/amazon/awssdk/crt/test/TlsContextOptionsTest.java

```diff
@@ -224,7 +224,7 @@ public void testMtlsPkcs11() {
         Assume.assumeTrue(System.getProperty("NETWORK_TESTS_DISABLED") == null);
         Pkcs11LibTest.assumeEnvironmentSetUpForPkcs11Tests();
 
-        try (Pkcs11Lib pkcs11Lib = new Pkcs11Lib(Pkcs11LibTest.TEST_PKCS11_LIB);
+        try (Pkcs11Lib pkcs11Lib = new Pkcs11Lib(Pkcs11LibTest.TEST_PKCS11_LIB, Pkcs11Lib.InitializeFinalizeBehavior.STRICT);
```

Comment:
Add a comment here about how softhsm (give version if you can) was crashing on platform (give exact config if you can) on shutdown if it wasn't finalized, so we can only test the STRICT config

and if Musl only allows a library to be unloaded once, mention that too, because it's really going to bite someone in the butt if they try to add more tests